src/containers/ProfileApiKeys/ProfileApiKeys.pcss

```diff
@@ -0,0 +1,25 @@
+td {
+  form {
+    .custom-control {
+      .custom-switch {
+        input {
+          top: 25px;
+        }
+      }
+
+      &-label {
+        &:before {
+          width: 2.75rem !important;
```

Comment:
why do you use rem instead of pixels?

Reply:
due to source code of react-bootstrap library